src/renderer/components/player/Player.tsx

```diff
@@ -17,6 +17,8 @@ const keyMap = {
   historyForward: ['Forward in time', 'right'],
   navigateBack: ['Go back to scene details', 'backspace'],
   toggleFullscreen: ['Toggle fullscreen', 'CommandOrControl+F'],
+  alwaysOnTop: ['Toggle On Top', 'CommandOrControl+T'],
+  toggleMenuBarDisplay: ['Show/Hide Menu', 'CommandOrControl+M'],
```

Comment:
Please choose another shortcut. Command+M on Mac is Minimize and can't be overridden.

Reply:
I changed it to ComandOrControl + Shift + 6  - and have checked it in - do I need to make a new PR?

Reply:
If you commit it to your origin/master then it should show up here 🤔 (Make sure it's pushed)

Reply:
No need to make a new PR 👍